docs/LedStrip.md

```diff
@@ -89,6 +89,7 @@ It could be possible to be able to specify the timings required via CLI if users
   * Fits well under motors on mini 250 quads.
 * [Adafruit NeoPixel Stick](https://www.adafruit.com/products/1426) (works well)
   * Measured current consumption in all white mode ~ 350 mA.
+* [Aliexpress SK6812 RBGWW strip](https://www.aliexpress.com/wholesale?SearchText=rgbww+sk6812) 
```

Comment:
Do you have a more reliable source?

Reply:
https://www.adafruit.com/product/2869 could be a candidate